shared/common-adapters/back-button.desktop.js

```diff
@@ -20,7 +20,7 @@ export default class BackButton extends Component {
     return (
       <div style={{...styles.container, ...this.props.style}} onClick={e => this.onClick(e)}>
         <Icon type='fa-arrow-left' style={styles.icon} />
-        <Text type='BodyPrimaryLink' onClick={e => this.onClick(e)}>Back</Text>
+        <Text type='BodyPrimaryLink' onClick={e => this.onClick(e)}>{this.props.title || 'Back'}</Text>
```

Comment:
the desktop nav uses a back button by default just so we had a way to be able to get around before we had finalized designs. this means we have a back button on a bunch of screen which actually don't intend to have it there. in order to make it super clear this is for debugging only i changed it to 'debug back' and plumbed this here